pages/docs/adapters/adapter-overview.md

```diff
@@ -32,6 +32,7 @@ Wherever meaningful (license, maturity of the project, no other home), we host t
 | [Ateles (APES)](https://apes.osdn.io/pages/ateles) | [Univ. Siegen STS](https://www.mb.uni-siegen.de/sts/index.html) | [code](https://osdn.net/projects/apes/scm/hg/ateles) | Fluid-Acousting, Fluid-Fluid coupling | |
 | [DuMuX](https://dumux.org/) | [Alexander Jaust, University of Stuttgart](https://www.ipvs.uni-stuttgart.de/institute/team/Jaust-00001/) | [code](https://git.iws.uni-stuttgart.de/dumux-appl/dumux-precice) | Free flow and porous-medium flow | Currently only coupled DuMuX to DuMuX. Feedback appreciated. |
 | [DUNE](https://dune-project.org/) | [Max Firmbach, UniBW M](https://www.unibw.de/imcs/team/firmbach) | [Thesis](https://mediatum.ub.tum.de/node?id=1609293), [code](https://github.com/precice/dune-adapter) | Structure part in FSI | |
+| [DUNE-Fem](https://www.dune-project.org/sphinx/content/sphinx/dune-fem/) | [Niklas Kotarsky, Lund University](https://www.lunduniversity.lu.se/lucat/user/9a5a021777b3e7cb0b8aea7ee9094808) | [coupled example code](https://github.com/precice/tutorials/tree/develop/flow-over-heated-plate/solid-dunefem) | Structure part in CHT, any FEM | |
```

Comment:
Should we point to the tutorial code (and to the `develop` branch), or to the website?

Reply:
Right, the `develop` should become a `master` eventually. Not sure what is better, code or website. I could imagine that people clicking here want to see code?

Reply:
Same here, just checking. Let's leave it like this for now.